src/main/java/com/selfxdsd/storage/SelfContributors.java

```diff
@@ -121,6 +154,7 @@ public Contributors ofProject(
         if (project == null) {
             return new EmptyContributors(this);
         }
+        final Page page = super.current();
```

Comment:
@amihaiemil is there a way to make ofProject faster like using just sql like grouping and such?

Reply:
@criske to be honest, I have no idea : )

Reply:
@amihaiemil When I have time I'll try to make it work as a single sql query, just as an exercise to learn jooq :D